client/config/config.go

```diff
@@ -48,22 +48,22 @@ func (c *ClientConfig) SetBroadcastMode(broadcastMode string) {
 
 // ReadFromClientConfig reads values from client.toml file and updates them in client Context
 func ReadFromClientConfig(ctx client.Context) (client.Context, error) {
+	homeFileDir := filepath.Join(ctx.HomeDir, "config")
+	homeFilePath := filepath.Join(homeFileDir, "home.toml")
```

Comment:
What is `home.toml`??

Reply:
It's a file that contains the path to the currently set home directory, so you don't have to pass `--home path/to/dir` to every single command when using a non-standard home. For local testing of a SDK-based chain this is the biggest drawback to DevEx IMO.

Do you have any objections to adding this?

Reply:
We already have a dedicated file for saved configurations -- `client.toml`

Reply:
@alexanderbez say you have 3 node configurations for one binary, you may want to define which home it will be used by default, without having to specify --home every time. Storing the default value home value at ~/.simapp/config/client.toml would be confusing, especially if you are not using the ~/.simapp home at all.
Personally, I found that home.toml making sense, as it should exist only once, at ~/.appd.

Reply:
@alexanderbez What do you think about this approach now that we've elaborated on it further? I would be reluctant to address any open comments on here if it's not clear that it will actually be merged.

Reply:
this sounds like a different tool is being built into the sdk to me. I think this should be a hubl concern not an sdk concern. adding more tomls than what is necessary to operate a node seems like a design smell to me. Hubls aim is to not have to do this sort of thing going forward.

Reply:
After reading up on `hubl` I didn't fully understand how that is going to replace the functionality proposed here to be honest, could you explain @tac0turtle? Seems like a useful tool definitely, but addressing something different it seems.

_Imagine the following use-case_

During development of a bigger chain feature, I'm running a local node based in `~/.appd`. I deploy a bunch of smart contracts to the chain, set up a keys, authorizations, staking, and other things. I am planning to re-use the data produced on this chain instance for a longer period of time and hence do NOT want to overwrite the directory.

In the mean time, there's another TODO to be addressed, which still would be best to be tested out manually. I go ahead and create a folder `~/.tmp-appd`, where I am running the testing node. During testing I find myself wanting to re-use some bash scripts that I use for setting up e.g. staking or authorizations. The only way that I can use this script is to plaster the whole thing with home flags added to every single command and then adjusting e.g. a variable `$HOMEDIR` in it. It would be much nicer if one could just configure the home directory, which the calls to `appd` will target beforehand and then just run the script.. Additionally, when I use some manual commands in the terminal it turns simple and concise commands like 

```
appd q bank balances $(appd keys show -a mykey)
```
 to 
```
appd q bank balances $(appd keys show -a mykey --home $HOME/.tmp-appd) --home $HOME/.tmp-appd
```

Is `hubl` a tool that could alleviate this?

Reply:
Or you could just an an environment variable...which seems like the idiomatic systems approach. 

Having to maintain a single file just to store which home directory to use seems backwards to me personally.

Reply:
@alexanderbez  Having an environment variable that handles this would be great, too! I'm really just looking for a way to get rid of having to carry this flag for every single command.

Something to keep in mind here is, that there would be an individual environment variable for every chain being run (or where a user wants to specify a default directory). Something like `APPD_HOME` being derived from the app name would work?

Reply:
Yeah you can name the env var whatever you like :)

But also keep in mind, I'm not rejecting or denying this proposal! If the team thinks this is a good idea, then lets proceed!